lib/relay/client/static.go

```diff
@@ -38,6 +39,8 @@ func newStaticClient(uri *url.URL, certs []tls.Certificate, invitations chan pro
 
 		messageTimeout: time.Minute * 2,
 		connectTimeout: timeout,
+
+		token: uri.Query().Get("token"),
```

Comment:
Just being cautious, are we sure this doesn't panic if you don't have a query string/query param?

Reply:
I just tested it but also the documentation of [net/url](https://pkg.go.dev/net/url) reads 
> Query parses RawQuery and returns the corresponding values. It **silently discards malformed** value pairs.

and

> Get gets the first value associated with the given key. **If there are no values associated with the key, Get returns the empty string**.